### PR TITLE
fix: Add sort buffer size parameter

### PR DESCRIPTION
**Pull request by venky-wandb**

### CODE REVIEW

**Comment by jsbroks** [comment]
how did you pick this number? Do you know what the default value is?

**Comment by jsbroks** [comment]
@venky-wandb could you also do this for gcp

**Review by jsbroks — APPROVED** [review]

**Comment by elainaRenee** [comment]
Are we seeing this issue come up again? Should we make this a configurable input with a default and monitor how increasing this impacts this instance first?

**Comment by jsbroks** [comment]
Yeah more issues are coming up. This is a short term fix. 
@venky-wandb want to add it as a variables?
`database_sort_buffer_size`

**Comment by venky-wandb** [comment]
Sure, makes sense! will add it as a variable.

**Comment by venky-wandb** [comment]
@jsbroks @elainaRenee Updated and tested, wanna give this comment a quick 👍 once you're re-reviewed it?

File: modules/database/main.tf [hunk 1]
```diff
@@ -4,7 +4,7 @@ locals {
   master_password = random_string.master_password.result
 
   major_mysql_version  = "5.7"
-  aurora_mysql_version = "2.10.0"
+  aurora_mysql_version = "3.02.0"
```
Comment (jsbroks) [inline modules/database/main.tf:7]: Can you delete these lines, we dont use theme

**Comment by jsbroks** [comment]
This PR is included in version 1.5.2 :tada:

**Pull request merged**